tikv-configuration-file.md

```diff
@@ -369,8 +369,13 @@ Configuration items related to storage.
 
 ### `enable-ttl`
 
+> **Warning:**
+>
+> - The TTL feature is only available for the RawKV interface for now. You can only configure this feature when creating a new cluster because TTL uses different data formats in the storage layer.
+> - You **must not** modify this item on an existing cluster. Otherwise, TiKV reports errors when it starts.
+> - You **must not** use this item on a cluster with TiDB nodes. Otherwise, serious consequences might occur, such as data corruption and the upgrade failure of TiDB clusters.
```

Comment:
```suggestion
> - **DO NOT** modify this configuration item on an existing cluster. Otherwise, TiKV reports errors when it starts.
> - **DO NOT** use this item on a cluster with TiDB nodes. Otherwise, serious consequences might occur, such as data corruption and the upgrade failure of TiDB clusters.
```